umpf

```diff
@@ -394,7 +394,7 @@ find_branch_rev() {
 				local choice=""
 				read -e -i n -p "Use ${branch} instead? [y/n]: " choice
 				if [ "${choice}" == "y" ]; then
-					reply="${branch}"
+					reply="$(${GIT} rev-parse "${branch}^{}" 2> /dev/null)"
```

Comment:
```suggestion
					reply="${b}"
```
No need to do the `rev-parse` again.

Reply:
sounds good. change incorporated and commit message adjusted, thanks.